### PR TITLE
Data Mining documentation

### PR DESCRIPTION
**Pull request by pavolloffay**
Hawkular Data Mining module documentation

### CODE REVIEW

**Comment by jkremser** [comment]
The page is not reachable w/o knowing the direct link.

File: src/main/jbake/content/docs/components/datamining/index.adoc [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ - Simple moving average
+ - Weighted moving average
+
+All variants of exponential smoothing contains optimizer which finds the best smoothing parameters for given training
+data set. Optimizers minimizes mean squared error of one step ahead prediction using non linear optimization
+algorithm (Nelder-Mead simplex).
+
+AutomaticForecaster can be used for the best model selecting. It selects the best model based on Akaike information
+criterion (AIC and AICc) or Bayesian information criterion (BIC).
+If the time series changes during time AutomaticForecaster is able to select different model than previously selected,
```
Comment (jkremser) [inline src/main/jbake/content/docs/components/datamining/index.adoc]: time AutomaticForecaster -> time, AutomaticForecaster

**Comment by jkremser** [comment]
lgtm

**Comment by pavolloffay** [comment]
It has do not merge yet tag .  I will send another PR

**Comment by jkremser** [comment]
ouch, sorry, I missed it.. I can revert if you want but it looks good. I mean, it doesn't look unfinished.

**Comment by pavolloffay** [comment]
It's fine I will send another PR, more cosmetic changes..

**Pull request merged**